src/satosa/base.py

```diff
@@ -285,3 +286,16 @@ def init_config(self, config):
     def expose_entityid_endpoint(self):
         value = self.config.get(self.KEY_ENTITYID_ENDPOINT, False)
         return bool(value)
+
+
+class SAMLEIDASBaseModule(SAMLBaseModule):
```

Comment:
It starts to feel a bit strange having all sorts of XBaseModules in base.py. Would it be an option to have a module.py where we define all BaseModules our {backends,frontends} inherit from ?